.golangci.yml

```diff
@@ -4,7 +4,7 @@ linters:
     - deadcode
     - depguard
     - dogsled
-    - dupl
+    # - dupl
```

Comment:
Is it an updated .golangci.yml for cosmos 44 or did you comment them yourself?

Reply:
Some of them I've commented cause of low severity